client/my-sites/site-settings/form-jetpack-scan.jsx

```diff
@@ -282,10 +282,10 @@ module.exports = React.createClass( {
 					disabled={ this.disableForm() } />
 
 				<FormSettingExplanation>
-					{ this.translate( 'Password is not required if you use public key authentication.' ) }
-					<a onClick={ this._onShowDialog } href="#">
-						{ this.translate( 'Get your public key string.' ) }
-					</a>
+					{ this.translate(
```

Comment:
Thanks, good eye.

This one is hard to test because I think the Security Scan is turned off in local environment.